package.json

```diff
@@ -117,7 +118,7 @@
     "babel-preset-es2015": "7.0.0-alpha.18",
     "babel-preset-es2016": "7.0.0-alpha.18",
     "babel-preset-es2017": "7.0.0-alpha.18",
-    "babel-preset-env": "2.0.0-alpha.18",
+    "babel-preset-env": "2.0.0-beta.2",
```

Comment:
if we use beta.2 in env but not everywhere else it will be weird or just not work?

Reply:
Yeah can we upgrade everything to the beta releases?